### PR TITLE
Add links to the API documentation to the downloads page

### PR DESCRIPTION
**Pull request by sbesson**
The API documentation is now synced to `downloads.openmicroscopy.org/{omero,bio-formats}/{$version}/api` as part of the release job process. 
This commit adds some links to this API documentation from the corresponding downloads page.

/cc @hflynn

### CODE REVIEW

**Comment by sbesson** [comment]
PR built as part of http://ci.openmicroscopy.org/view/Release/job/OMERO-4.4-download-page/82 and http://ci.openmicroscopy.org/view/Release/job/BIOFORMATS-4.4-download-page/31. See the corresponding output at http://downloads.openmicroscopy.org/omero/4.4.10-rc1/ and http://downloads.openmicroscopy.org/bio-formats/4.4.10-rc1/.

File: omero_downloads.html [hunk 1]
```diff
@@ -257,7 +257,10 @@
 
 <ul>
   <li>
-<p>Other versions of the API documentation are available <a href="artifacts">here</a></p>
+  <p>The OMERO API documentation is available <a href="api">here</a></p>
```
Comment (hflynn) [inline omero_downloads.html]: Replace with:
"The OMERO API documentation is also available <a href="api">unzipped</a>"

Comment (hflynn) [inline omero_downloads.html]: Or
unzipped -> to read on the web
?

File: omero_downloads.html [hunk 2]
```diff
@@ -257,7 +257,10 @@
...
+  </li>
+  <li>
+  <p>Other versions of the API documentation are available <a href="artifacts">here</a></p>
```
Comment (hflynn) [inline omero_downloads.html]: I'd delete this in favour of a note to say that "Zipped files of API documentation for other versions of Ice are available as part of the build artifacts (see link below)"

**Comment by hflynn** [comment]
BF is fine though :+1:

**Comment by hflynn** [comment]
Looks good - we are now introducing links with just 'here' as the text but I guess as these are link checked by the build rather than plone now, it doesn't really matter.

**Comment by sbesson** [comment]
@hflynn: sadly there is no link check step in the downloads page generation (not generated by Sphinx). Downloads pages above have been regenerated with the correct wording.

**Comment by joshmoore** [comment]
Merging for RC2. If there are any further changes wanted/needed, let's go for another PR.

**Comment by sbesson** [comment]
--rebased-to #35

**Comment by sbesson** [comment]
--rebased-to #36

**Pull request merged**